server/models.py

```diff
@@ -926,6 +926,38 @@ class ActivityLogRecord(Base):
     posted_to_slack_at = Column(UTCDateTime)
 
 
+class BatchInventoryData(BaseModel):
+    jurisdiction_id = Column(
+        String(200),
+        ForeignKey("jurisdiction.id", ondelete="cascade"),
+        nullable=False,
+        primary_key=True,
+    )
+
+    cvr_file_id = Column(String(200), ForeignKey("file.id", ondelete="set null"))
+    cvr_file = relationship(
+        "File",
+        foreign_keys=[cvr_file_id],
+        single_parent=True,
+        cascade="all, delete-orphan",
+    )
+    tabulator_status_file_id = Column(
+        String(200), ForeignKey("file.id", ondelete="set null")
+    )
+    tabulator_status_file = relationship(
+        "File",
+        foreign_keys=[tabulator_status_file_id],
+        single_parent=True,
+        cascade="all, delete-orphan",
+    )
+
+    election_results = Column(JSON)
+    tabulator_id_to_name = Column(JSON)
```

Comment:
I'm always a little bit hesitant to use JSON fields instead of more structured fields. But I take it we're doing this to get something up and running quickly

Reply:
Yeah, I imagine this will likely change in the future, so sticking to quick and dirty for now.